scripts/tsd.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -xe
+
+cp ./test/types/* ./build/
```

Comment:
We don't need to include tests (types) in the build.

Reply:
Right.  Done